data/experiment-recipe-samples/android-96-multifeature.json

```diff
@@ -0,0 +1,87 @@
+{
+  "appId": "org.mozilla.firefox",
+  "appName": "fenix",
+  "application": "org.mozilla.firefox",
+  "arguments": {},
+  "branches": [
+    {
+      "features": [
+        {
+          "enabled": false,
+          "featureId": "fenix-default-browser",
+          "value": {}
+        },
+        {
+          "enabled": true,
+          "featureId": "homescreen",
+          "value": {
+            "sections-enabled": {
+              "jumpBackIn": false,
+              "pocket": false,
+              "recentExplorations": false,
+              "recentlySaved": false,
+              "topSites": true
+            }
+          }
+        }
+      ],
+      "ratio": 1,
+      "slug": "control"
+    },
+    {
+      "features": [
+        {
+          "enabled": true,
+          "featureId": "fenix-default-browser",
+          "value": {}
+        },
+        {
+          "enabled": true,
+          "featureId": "homescreen",
+          "value": {
+            "sections-enabled": {
+              "jumpBackIn": true,
+              "pocket": true,
+              "recentExplorations": true,
+              "recentlySaved": true,
+              "topSites": true
+            }
+          }
+        }
+      ],
+      "ratio": 1,
+      "slug": "treatment"
```

Comment:
For the different treatments of the "fenix-default-browser" feature, the branch name seems to matter.